sphinx/source/docs/user_guide/interaction/callbacks.rst

```diff
@@ -77,10 +77,57 @@ is executed to update some data:
     :source-position: above
 
 
+
+CustomJS for Event classes
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In addition to responding to Backbone events using ``js_on_change``,
```

Comment:
Maybe:

In addition to responding to property change events using ``js_on_change``, Bokeh allows 
CustomJS callbacks to be triggered by specific interaction events with the 
plot canvas, on button click events, and LOD events.

These event callbacks are defined on models using the ``js_on_event`` method,
with the callback receiving the event object as a locally defined ``cb_obj``
variable.